readme.md

```diff
@@ -174,6 +174,31 @@ const memGot = mem(got, {cache});
 })();
 ```
 
+### WeakMap cache with multiple arguments
+
+If you want to memoize a function of multiple object arguments, you can use [deep-weak-map](https://github.com/futpib/deep-weak-map) as a replacement for the default cache.
```

Comment:
```suggestion
If you want to memoize a function with multiple object arguments, you can use [deep-weak-map](https://github.com/futpib/deep-weak-map) as a replacement for the default cache.
```

Reply:
Can you include something about what problem your module solves? Like, how does it solve the problem of memoizing multiple object arguments? And how does it do it differently from, for example, `JSON.stringify()` or `serialize-javascript` which are mentioned elsewhere in the docs.